8.x/theme-development/helpers.md

```diff
@@ -683,221 +964,280 @@ This function takes the parent directory and the number of columns you want to i
+                        <ul>
+                            @foreach($directory->directories as $sub_directory)
+                                <li>
+                                    {{$sub_directory->title}}
+                                </li>
+                            @endforeach
+                        </ul>
+                    </li>
+                @endforeach
+            </ul>
+        </li>
     @endforeach
 </ul>
 ```
 
-Output
-```
-Latest important products for directory: ظروف سرو رومیزی
-کاسه قطر ۱۶cm کرم بانددار
-بشقاب تخت اوال قطر ۲۵cm قرمز بانددار
-تخته ظروف سرو چدنی ۲۲×۳۵.۵ سانتیمتر
-شات کوکتل ۱۰۰ml مارتینی لیبی
-بورج ۳۵۵ml جیبرالتر لیبی
-تنگ ۲۰۰ml بایجو لیبی
-دیگچه چکشی ۲۰۰۰ml
-بشقاب گرد ۲۴ سانتی فیوژن RAK زرد
-ظرف سرو قیفی سیب زمینی ۲ تایی مشکی
-استند باربیکیو ناندوز به همراه ۶ عدد سیخ
-```
-________________________________
+The output would show the directories and subdirectories as a list based on your system data. Here is an output sample:
 
+```html
+
+<ul>
+    <li>
+        <ul>
+            <li>
+                <h4>Directory 1</h4>
+                <ul>
+                    <li>Subdirectory 1.1</li>
+                    <li>Subdirectory 1.2</li>
+                    <li>Subdirectory 1.3</li>
+                </ul>
+            </li>
+            <li>
+                <h4>Directory 2</h4>
+                <ul>
+                    <li>Subdirectory 2.1</li>
+                    <li>Subdirectory 2.2</li>
+                    <li>Subdirectory 2.3</li>
+                </ul>
+            </li>
+        </ul>
+    </li>
+    <li>
+        <ul>
+            <li>
+                <h4>Directory 3</h4>
+                <ul>
+                    <li>Subdirectory 3.1</li>
+                    <li>Subdirectory 3.2</li>
+                    <li>Subdirectory 3.3</li>
+                </ul>
+            </li>
+            <li>
+                <h4>Directory 4</h4>
+                <ul>
+                    <li>Subdirectory 4.1</li>
+                    <li>Subdirectory 4.2</li>
+                    <li>Subdirectory 4.3</li>
+                </ul>
+            </li>
+        </ul>
+    </li>
+</ul>
+```
 
+---
+#### get_directory
 
-### `get_visible_product_leaves`
 ```php
-function get_visible_product_leaves(Directory $root_directory, int $count): array|Collection
+function get_directory($directory_id): ?Directory {
+        return Directory::find($directory_id);
+    }
+
 ```
-Takes the directory and sorts the specified number of product_leaves that have the visible store feature active.
+This function takes `directory_id` and returns the directory title. The output will be `null` if it does not find the directory.
+
+
+Example:
 
 ```php
-<h2>Latest visible products for directory: {{$directory->title}}</h2>
+<p> The directory with  id `2` has the title of `{{get_directory(2)->title}}`</p>
+```
+
+Output:
+```html
+<p> The directory with id `2` has the title of `< the name of the directory with id '2'>` </p>
```

Comment:
< the name of the directory with id '2'>  ->  <the-title-of-the-directory-with-id-'2'>